mqtt/mqttd/src/broker/bootstrap/edgehub.rs

```diff
@@ -73,9 +70,7 @@ pub async fn broker(
     config: &BrokerConfig,
     state: Option<BrokerSnapshot>,
 ) -> Result<Broker<impl Authorizer>> {
-    let device_id = env::var(DEVICE_ID_ENV)?;
-
-    let authorizer = LocalAuthorizer::new(EdgeHubAuthorizer::new(PolicyAuthorizer::new(device_id)));
+    let authorizer = LocalAuthorizer::new(EdgeHubAuthorizer::new(AllowAll));
```

Comment:
If you want to enable all authorizer, could you please add a TODO comment and make sure we have a work item to track this. Otherwise, this can lead to a security vulnerability down the road if unattended.

Reply:
Vadim's going to reactivate it in his next pr that sends over policy to edgeHub. 
Added todo